lib/normalize.js

```diff
@@ -44,6 +44,9 @@ module.exports = {
    * All calculations assume GMT.
    */
   date: function(str, inFormat, outFormat) {
-    return moment.utc(str, inFormat).format(outFormat || 'YYYY-MM-DD');
+    if (str) {
+      return moment.utc(str, inFormat).format(outFormat || 'YYYY-MM-DD');
+    }
+    return str;
```

Comment:
I think it was just a cheap way to avoid the moment error when trying parse empty values.  I'd leave this as-is for now and deal with it next time it proves to be an issue.